config/core/resources/channel.yaml

```diff
@@ -207,6 +207,13 @@ spec:
                 description: Annotations is additional Status fields for the Resource to save some additional State as well as convey more information to the user. This is roughly akin to Annotations on any k8s resource, just the reconciler conveying richer information outwards.
                 type: object
                 x-kubernetes-preserve-unknown-fields: true
+              auth:
+                description: Auth provides the relevant information for OIDC authentication.
+                type: object
+                properties:
+                  serviceAccountName:
+                    description: ServiceAccountName is the name of the generated service account used for this components OIDC authentication.
+                    type: string
```

Comment:
@karthikmurali60 this seems to be the wrong place. You added it correctly in `pkg/apis/duck/v1/subscribable_types.go` in the `SubscriberStatus`. But the SubscriberStatus is in `.status.subscribers`: https://github.com/knative/eventing/blob/347409f3f1956d85f95cec7bcef5fa2b2edc306d/config/core/resources/channel.yaml#L286-L304